themes/digital.gov/layouts/guides/list.html

```diff
@@ -0,0 +1,97 @@
+{{/* Display guides in a card list that uses the guide_weight field to display each card */}}
+
+{{ define "content" }}
+
+  {{ $guidesList := slice }}
+  {{ $guidesData := slice }}
+  {{ $guidePages := slice }}
+
+  {{/* Build list of guide names from yaml files in string format */}}
+  {{ range $i, $names := $.Site.Data.guidenav }}
+    {{ $guidesList = $guidesList | append $i }}
+  {{ end }}
+
+  {{/* Iterate over list of guide names and save each as a map of strings */}}
+  {{ range $guidesList }}
+    {{ $guidesData = $guidesData | append (index $.Site.Data.guidenav .) }}
+  {{ end }}
+
+  {{/* Save each guide homepage as hugo page collection */}}
+  {{ range $guidesData }}
+    {{/* Get the first nav item from each guide, the guide homepage */}}
+    {{ range first 1 .nav }}
+      {{ $guidePages = $guidePages | append ($.Site.GetPage .path) }}
+    {{ end }}
+
+    {{/* If guide is an HCD sub-guide, get the parent url which is the HCD home page */}}
+    {{ if .landingURL }}
+      {{ $guidePages = $guidePages | append ($.Site.GetPage .landingURL) }}
+    {{ end }}
+
+    {{/* Remove any duplicates from the previous step */}}
+    {{ $guidePages = uniq $guidePages }}
+  {{ end }}
+
+
+  <main role="main" id="main-content">
+    <div class="grid-container-desktop margin-bottom-5">
```

Comment:
## Future enhancement

Setting a margin on every landing page could be hard to maintain. We'd have the same current problem where you need to update each template manually.

It'd be a better pattern to create a more generic class that holds the page content.

Reply:
Made a card to capture this.